docs/dev.rst

```diff
@@ -1,20 +1,522 @@
+
+
+
+Reference Unmanaged Projects
+****************************
+
+If the referenced repository does not have a namespace configured, or if the dependency specifies the ``unmanaged`` option as ``True``, the repository is treated as an unmanaged repository.
+
+    Example: Salesforce EDA
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - github: https://github.com/SalesforceFoundation/EDA
+              unmanaged: True
+
+..
+
+    The EDA repository is configured for a namespace, but the dependency  specifies ``unmanaged: True``, so the dependency deploys unmanaged EDA and its dependencies.
+
+
+
+Reference a Specific Tag
+************************
+
+To reference a version other than HEAD and the latest production release, use the ``tag`` option to specify a tag from the target repository. This option is most useful for testing against beta versions of underlying packages, or recreating specific org environments for debugging.
+
+    Example: Salesforce EDA
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - github: https://github.com/SalesforceFoundation/EDA
+              tag: beta/1.47-Beta_2
+
+..
+
+    The EDA repository's tag, ``beta/1.47-Beta_2``, is used instead of the latest production release of EDA (1.46, for this example). This tag lets a build environment use features in the next production release of EDA that are already merged but not yet included in a production release.
+
+
+
+Skip ``unpackaged/*`` in Reference Repositories
+***********************************************
+
+If the repository you are referring to has dependency metadata under unpackaged/pre or unpackaged/post, use the ``skip`` option to skip deploying that metadata with the dependency.
+
+    Example: Salesforce EDA
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - github: https://github.com/SalesforceFoundation/EDA
+              skip: unpackaged/post/course_connection_record_types
+
+
+Automatic Cleaning of ``meta.xml`` Files on Deploy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To let CumulusCI fully manage the project's dependencies, the ``deploy`` task (and other tasks based on ``cumulusci.tasks.salesforce.Deploy``, or subclasses of it) automatically removes the ``<packageVersion>`` element 
+and its children from all ``meta.xml`` files in the deployed metadata. Removing these elements does not affect the files on the filesystem.
+
+The reason for stripping ``<packageVersion>`` elements on deploy is that the target Salesforce org automatically adds them back with the installed version of the referenced namespace. This feature lets CumulusCI fully manage dependencies, and avoids rushing a new commit of ``meta.xml`` files when a new underlying package version is available.
```

Comment:
Am I correct in changing it to "this feature"?

Reply:
Works for me.